js/server.envirophat.ocf.js

```diff
@@ -0,0 +1,184 @@
+/** Copyright 2017 Vprime
```

Comment:
`git rm`

Reply:
I see what happened. I did not stage removal of those files. it should be good now